cmake/EthCompilerSettings.cmake

```diff
@@ -144,7 +144,13 @@ else ()
 endif ()
 
 if (SANITIZE)
-	set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fno-omit-frame-pointer -fsanitize=${SANITIZE}")
+	# Perform case-insensitive string compare
+	string(TOLOWER "${SANITIZE}" san)
+	# -fno-omit-frame-pointer gives more informative stack trace in case of an error
+	# -fsanitize-address-use-after-scope throws an error when a variable is used beyond its scope
```

Comment:
Sounds very useful!

Reply:
I created another issue to discuss additional runtime (not compilation) flags we can pass to ASan https://github.com/ethereum/solidity/issues/6813